Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM alpine:latest
+MAINTAINER Erik Osterman "erik@cloudposse.com"
+
+USER root
+
+ARG OPENSSH_VERSION=V_7_1_P1
+
+RUN apk --update add linux-pam libssl1.0 ca-certificates openssl && \
+    update-ca-certificates && \
+    ln -s /lib /lib64
+
+RUN apk add --virtual .build-deps build-base automake autoconf libtool git linux-pam-dev openssl-dev wget && \
+    mkdir -p /usr/src && \
+    cd /usr/src && \
+    ( wget https://dl.duosecurity.com/duo_unix-latest.tar.gz && \
+      tar zxf duo_unix-latest.tar.gz && \
+      cd duo_unix-* && \
+      ./configure --with-pam --prefix=/usr && \
+      make && \
+      make install && \ 
+      cd .. && \
+      rm -rf duo_unix-* && \
+      rm -f duo_unix-latest.tar.gz \
+    ) && \
+    ( git clone https://github.com/google/google-authenticator-libpam /usr/src/google-authenticator-libpam && \
+      cd /usr/src/google-authenticator-libpam && \
+      ./bootstrap.sh && \
+      ./configure --prefix=/ && \
+      make && \
+      make install) && \
+    ( git clone https://github.com/openssh/openssh-portable.git /usr/src/openssh && \
+      cd /usr/src/openssh && \
+      git checkout ${OPENSSH_VERSION} && \
+      sed -i -e '/_PATH_XAUTH/s:/usr/X11R6/bin/xauth:/usr/bin/xauth:' pathnames.h && \
+      sed -i -E 's/OpenSSH_[0-9.]+/SERVER/' version.h && \
+      autoreconf && \
+      ./configure \
+          --prefix=/usr \
+          --sysconfdir=/etc/ssh \
+          --datadir=/usr/share/openssh \
+          --libexecdir=/usr/lib/ssh \
+          --mandir=/usr/share/man \
+          --with-pid-dir=/run \
+          --with-mantype=man \
+          --with-privsep-path=/var/empty \
+          --with-privsep-user=sshd \
+          --with-md5-passwords \
+          --with-ssl-engine \
+          --with-cflags="-D _FORTIFY_STDLIB_H" \
```

Comment:
@goruha I need your help to avoid this hack. I can't get OpenSSH to compile on Alpine without it. This hack disable's fortification on `stdlib.h` functions (e.g. `realpath`). 

See: https://github.com/neochrome/docker-bastion/issues/1